wp-content/civi-extensions/goonjcustom/Civi/DroppingCenterService.php

```diff
@@ -78,6 +192,12 @@ public static function droppingCenterTabset($tabsetName, &$tabs, $context) {
       return;
     }
     $tabConfigs = [
+      'SendDispatchEmail' => [
+        'title' => ts('Send Dispatch Email'),
+        'module' => 'afformSendDispatchEmail',
+        'directive' => 'afform-send-dispatch-email',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCampService.tpl',
+      ],
```

Comment:
This is a new tab for sending the Dispatch Email, where a form builder button needs to be added. The requirement is to have a button for sending the Dispatch Email available whenever the Goonj team needs it.